app/services/unified_chat_service.py

```
@@ -1437,6 +1438,182 @@ def _get_placeholder_opportunities(
+
+        try:
+            parsed_id = uuid.UUID(str(user_id))
+        except (ValueError, TypeError):
+            parsed_id = user_id
+
+        try:
+            async with get_database_session() as db:
+                result = await db.execute(select(User.role).where(User.id == parsed_id))
+                role = result.scalar_one_or_none()
+                is_admin = role == UserRole.ADMIN
+        except Exception as exc:  # pragma: no cover - defensive guard
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)